isso/ext/notifications.py

```diff
@@ -3,16 +3,20 @@
 from __future__ import unicode_literals
 
 import sys
-import io
 import time
 import json
+import os.path
 
 import socket
 import smtplib
 
 from email.utils import formatdate
 from email.header import Header
+from email.mime.multipart import MIMEMultipart
 from email.mime.text import MIMEText
+from jinja2 import Environment, FileSystemLoader
+import jinja2.exceptions as jinja2_exceptions
+from translate import Translator
```

Comment:
Can we please do this without an extra dependency? All that's needed is a dictionary.

Reply:
As far as I can tell this will actually make HTTP calls.